### ISSUE TITLE
Heading label is not translatable via WPML plugin

### ISSUE THREAD

**Issue by ankit644** [ahmedkaludi/easy-table-of-contents#415]
Ref ticket: https://magazine3.in/conversation/98838?folder_id=11

**Comment by viliusle** [comment]
That is true, it comes from WP admin (DB) and is not translatable by default. I had same problem. 

You could add support by adding this line in plugin: `$toc_title = __($toc_title);` (not recommended) 
or
use filter `ez_toc_get_option_heading_text` to catch it and translate the way you want in your theme functions.php for example:
`add_filter('ez_toc_get_option_heading_text', 'ez_toc_get_option_heading_text_handler');`

Hope it helps others.
